### ISSUE TITLE
Question: Loading app config connection string from Keyvault instead of hard coding

### ISSUE THREAD

**Issue by Karthickramk** [microsoft/spring-cloud-azure#714]
Currently in our application, we have hard coded the connection string in bootstrap.yaml. Instead of doing that can we configure it in the key vault and reference it in the bootstrap.yaml?

**Comment by saragluna** [comment]
Thanks for reaching out. Is it the connection string for App Configuration you're referring to?

**Comment by Karthickramk** [comment]
Yeah. App configuration connection string to be referred from keyvault.

**Comment by mrm9084** [comment]
@Karthickramk Are you hardcoding it or have a placeholder value? I would suggest moving to another form of authentication. Depending on your setup you could use, Managed Identity or a Service Principal to authenticate, or any other form supported by the [Azure SDK for Java](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/identity/azure-identity#credentials).

**Comment by chenrujun** [comment]
Closing this issue.
Because it's not active for a long time.
If anyone have similar issue, please create issue in new repo: https://github.com/Azure/azure-sdk-for-java/issues
